cmd/containerd-shim/main.go

```diff
@@ -38,6 +38,11 @@ func main() {
 			Name:  "debug",
 			Usage: "enable debug output in logs",
 		},
+		cli.StringFlag{
+			Name:  "socket, s",
+			Usage: "socket path for containerd-shim's GRPC server",
+			Value: "/run/containerd/containerd-shim.sock",
```

Comment:
`shim.sock`

This would break running multiple containers at the same time otherwise as they would all use the same socket by default.